### ISSUE TITLE
"authenticated-read" canned ACL is not supported

### ISSUE THREAD

**Issue by evgeniiz321** [nspcc-dev/neofs-s3-gw#858]
Due to https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html (Canned ACL section)

> authenticated-read - Owner gets FULL_CONTROL. The AuthenticatedUsers group gets READ access.

Currently we get for bucket acls - 

```
{'Grantee': 
{'DisplayName': '', 'EmailAddress': '', 'ID': '', 'Type': 'Group', 'URI': 'http://acs.amazonaws.com/groups/global/AllUsers'}, 'Permission': 'FULL_CONTROL'}, 
{'Grantee': {'DisplayName': '', 'EmailAddress': '', 'ID': '031a6c6fbbdf02ca351745fa86b9ba5a9452d785ac4f7fc2b7548ca2a46c4fcf4a', 'Type': 'CanonicalUser', 'URI': ''}, 'Permission': 'FULL_CONTROL'}
```

URI is not correct (it is http://acs.amazonaws.com/groups/global/AllUsers, should be http://acs.amazonaws.com/groups/global/AuthenticatedUsers) and Permission should be READ instead of FULL_CONTROL

Same applied for object ACLs as well.

**Comment by smallhive** [comment]
Part with the wrong permission FULL_CONTROL instead of READ was fixed in #857.

Groups, I'm not sure we may support this rule. For NeoFS, there exist only two groups: owner and others. We can't divide them into `AuthenticatedUsers` and `NotAuthenticatedUsers`. I mean, ACL rules from NeoFS give us only _owner_ and _others_ groups. We can't detect who is who from this information.

As I see there are no extra fields (in EACL table) to put this info to simplify detecting.
Did I miss something? @roman-khimov

**Comment by roman-khimov** [comment]
Likely can be documented, at least until we have any better ideas.
